src/commands/new/index.ts

```diff
@@ -0,0 +1,2 @@
+import file from './file.js';
```

Comment:
I think you can remove the `.js` extension:

```suggestion
import file from './file';
```